src/lib/gadgets/bitwise.unit-test.ts

```diff
@@ -1,7 +1,7 @@
 import { ZkProgram } from '../proof_system.js';
 import {
   Spec,
-  equivalent,
+  equivalentProvable as equivalent,
```

Comment:
Unrelated fix: Unit tests need `equivalentProvable` which tests implementations in a circuit